tasks/tool.py

```diff
@@ -63,12 +65,20 @@ def get_stack_name_prefix() -> str:
     return user_name.replace(".", "-")  # EKS doesn't support '.'
 
 
-def get_stack_json_outputs(full_stack_name: str) -> Any:
-    output = subprocess.check_output(
-        ["pulumi", "stack", "output", "--json", "-s", full_stack_name]
+def get_stack_json_outputs(ctx: Context, full_stack_name: str) -> Any:
+    buffer = StringIO()
+    global_flags = ""
+
+    # Checking root path
+    root_path = _get_root_path()
+    if root_path != os.getcwd():
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
You can achieve this by encapsulating the `ctx run` in ` with ctx.cd(root_path)`. I had to do a similar fix in the clean up task just seconds away https://github.com/DataDog/datadog-agent/pull/18052/files#diff-2e14236948e1a1fe87cd7e14b35b0beeb5bfa024b11f8e183e704ff37ab16a9fR155-R165

This works too by the way !

Reply:
I followed your suggestion and I think it's much clearer, thanks !